docs/spec_draft.md

```diff
@@ -376,6 +378,51 @@ specification. Numeric precision is implementation-defined.
 
 [Back to Ops](#index-of-documented-ops)
 
+## stablehlo.divide
+
+`stablehlo.divide(dividend, divisor) -> result`
+
+### Semantics
+
+Performs element-wise division of tensors `dividend` and `divisor` and produces
+a `result` tensor. For floating-point element types, implements the `division`
+operation from IEEE-754 specification. For n-bit integer types, division
+overflow (division by zero or division of $-2^{n-1}$ with $-1$) produces an
+implementation-defined value.
```

Comment:
If integer division needs rounding, which way does it go?

Reply:
I will rephrase both the divide and remainder specs to clarify what the they mean for integer types.